dev/entry.js

```diff
@@ -3,4 +3,32 @@ import mockPost from './mock-post.json'
 import React from 'react'
 import ReactDOM from 'react-dom'
 
-ReactDOM.render(<Article post={mockPost} />, document.getElementById('root'))
+ReactDOM.render(
+  <Article
+    post={mockPost}
+    elementColors={{
+      date: '#55e07f',
+      shape: '#73fa9c',
+      subTitle: '#7e33f6',
+      title: '#7e33f6',
+      topicName: '#7e33f6',
+      // body
+      annotation: '#55e07f',
+      blockQuote: '#494949',
+      blockQuoteBy: '#494949',
+      caption: '#494949',
+      infoboxContent: '#494949',
+      infoboxTitle: '#494949',
+      line: '#70f197',
+      link: '#55e07f',
+      metadataDefault: '#808080',
+      metadataPrimary: '#55e07f',
+      metadataSecodary: '#55e07f',
+      paragraph: '#494949',
+      quote: '#838383',
+      sectionSubtitle: '#494949',
+      sectionTitle: '#494949',
```

Comment:
Do we need to specify all the colors we are going to use here?

In mockup, there default color(like grey, black) are almost the same.

Could we only provide primary and secondary color?
```
<ThemeProvider theme={{
  color: {
    primary: {
      text: '#55e07f',
      line: '#70f197',
      shape: '#73fa9c',
    },  // primary set for green theme
    secondary: '#7e33f6', // purple
  },
}}
>
```

Reply:
Updated